src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -845,13 +845,58 @@ public void remove_rollback_directory_if_exists(string packageName)
             return packageInstalls;
         }
 
-        /// <summary>
-        /// Sets the configuration for the package upgrade
-        /// </summary>
-        /// <param name="config">The configuration.</param>
-        /// <param name="packageInfo">The package information.</param>
-        /// <returns>The original unmodified configuration, so it can be reset after upgrade</returns>
-        private ChocolateyConfiguration set_package_config_for_upgrade(ChocolateyConfiguration config, ChocolateyPackageInformation packageInfo)
+		public ConcurrentDictionary<string, PackageResult> get_outdated(ChocolateyConfiguration config)

```

Comment:
XML comments here might be helpful 👍

Reply:
They are lower with the method [Here](https://github.com/chocolatey/choco/pull/1495/commits/b16e17bab08600dd7f536cfb1fd0ff31a0c0e5bc?diff=unified#diff-03afb4ba12230f0f345a29a934727585R893) It just showed up oddly.

Reply:
I meant, it might be beneficial to add xml comments to this method. I also noticed the spacing looks funny, how many spaces over is that?

Reply:
These are 2 tabs, they look odd on github.

This is in INugetService
```
		/// <summary>
		///   Get outdated packages
		/// </summary>
		/// <param name="config">The configuration.</param>
		ConcurrentDictionary<string, PackageResult> get_outdated(ChocolateyConfiguration config);
```

Reply:
INuGetService, there we go!